src/components/content/Container.jsx

```diff
@@ -0,0 +1,28 @@
+import {Switch, Route} from "react-router-dom";
+import { useDispatch, useSelector } from 'react-redux';
+import {ContainerDiv} from './ContainerStyles'
+
+// Components
+import Hello from '../Hello';
+import {setLogged} from "../../actions/user";
+import ProfileIndex from "../profile/ProfileIndex";
+
+const Content = ({show}) => {
+  const dispatch = useDispatch()
+  const { isLogged } = useSelector(state => state.user)
+
+  return(
+    <ContainerDiv> {/*Div para styled-components*/}
+      <section className={show && 'close'}>
```

Comment:
Es mejor usar componentes propias de la libreria: https://mui.com/components/box/
```suggestion
      <Box className={show && 'close'} component="section">
```